installer/openSUSE/prepare

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+PKGEXT='rpm'
+DISTROAKA='openSUSE'
```

Comment:
let's keep non-user-visible references to openSUSE lowercased to avoid case-sensitivity issues.  That also includes the directory structure.